packages/cozy-harvest-lib/src/components/TriggerManager.jsx

```diff
@@ -410,7 +415,29 @@ DumbTriggerManager.propTypes = {
    * Whether the vault will be closable or not.
    * @type {Boolean}
    */
-  vaultClosable: PropTypes.bool
+  vaultClosable: PropTypes.bool,
+  /**
+   *
+   */
+  vaultClient: PropTypes.object,
+  client: PropTypes.object,
+  onError: PropTypes.func,
+  showUnlockForm: PropTypes.func,
+  onVaultDismiss: PropTypes.func,
+  error: PropTypes.any,
```

Comment:
why not using 
```
PropTypes.oneOfTypes([
PropTypes.string,
  PropTypes.object,
])``` ?

Reply:
Because adding / documenting something is better than nothing. If you prefer, next time, I'll not take the time to at least add the missing PropTypes.

Reply:
No problem! :-)